src/jackson.js

```diff
@@ -71,6 +79,16 @@ const validateApiKey = (token) => {
   return env.apiKeys.includes(token);
 };
 
+const extractAuthToken = (req) => {
```

Comment:
This should not be moved out of utils, it is also used by oauth.js for API key extraction

Reply:
Noted.